org/cloudfoundry.yml

```diff
@@ -2014,22 +1949,13 @@ orgs:
       routing-ci:
         archived: true
         has_projects: true
-      routing-concourse:
```

Comment:
Hi @geofffranks , with routing-concourse we thought about spinning up a VM for our WG similar to other WGs. We actually did not have capa to pick it up and do not yet know if we get capa in 24q3. If it does not hurt too much, you could keep it a bit longer, but we can also clean it up now and re-request once there is an actual plan+timeline.

Reply:
👍will keep this for you